# Older gcc compiler has incompatible deprecated attribute definition
# Avoid error: wrong number of arguments specified for 'deprecated' attribute

--- git-compat-util.h.orig	2022-10-07 06:48:26.000000000 +0200
+++ git-compat-util.h	2022-11-07 18:57:59.457282645 +0100
@@ -189,12 +189,7 @@
 #define _NETBSD_SOURCE 1
 #define _SGI_SOURCE 1
 
-#if defined(__GNUC__)
-#define UNUSED __attribute__((unused)) \
-	__attribute__((deprecated ("parameter declared as UNUSED")))
-#else
 #define UNUSED
-#endif
 
 #if defined(WIN32) && !defined(__CYGWIN__) /* Both MinGW and MSVC */
 # if !defined(_WIN32_WINNT)
